pagopa/io-app
# feat(Sicilia Vola): [IASV-42,IASV-43] Voucher details screen

## Short description
This PR add the voucher details screen and include the `VoucherInformationComponent` in a bottomsheet.
It also modify the navigation stack in order to avoid unexpected navigation.

## List of changes proposed in this pull request
- Add the `VoucherDetailsScreen` component
- Add the `VoucherDetailBottomSheet` component
- Refactor the navigation stack in order to separate the voucher list from the voucher generation navigation


https://user-images.githubusercontent.com/11773070/134663511-442e6198-619a-4cfb-a8ea-e70615c324a0.mp4



## How to test
Starts the flow from the SiciliaVola service detail press the Voucher List button then select an item.

Make sure to move to [this branch](https://github.com/pagopa/io-dev-api-server/tree/IASV-14-getPossibleVoucherState-api) on dev-server.

Note that the details data are stubbed in the api call handler so you receive always the same response.
